server/command/command.go

```diff
@@ -71,16 +78,22 @@ var ConfluenceCommandHandler = Handler{
 	defaultHandler: executeConfluenceDefault,
 }
 
-func GetCommand() *model.Command {
-	return &model.Command{
-		Trigger:          "confluence",
-		DisplayName:      "Confluence",
-		Description:      "Integration with Confluence.",
-		AutoComplete:     true,
-		AutoCompleteDesc: "Available commands: subscribe, list, unsubscribe \"<name>\", edit \"<name>\", install cloud/server, help.",
-		AutoCompleteHint: "[command]",
-		AutocompleteData: getAutoCompleteData(),
+func GetCommand(pAPI PluginAPI) (*model.Command, error) {
+	iconData, err := command.GetIconData(pAPI, "assets/icon-small.svg")
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get icon data")
 	}
+
+	return &model.Command{
+		Trigger:              "confluence",
+		DisplayName:          "Confluence",
+		Description:          "Integration with Confluence.",
+		AutoComplete:         true,
+		AutoCompleteDesc:     "Available commands: subscribe, list, unsubscribe \"<name>\", edit \"<name>\", install cloud/server, help.",
```

Comment:
While you are touching this line: Because there is autocomplete now we can drop the `"<name>"` part and also  `cloud/server`

Reply:
Do you also want me to remove it from this line as well?
https://github.com/mattermost/mattermost-plugin-confluence/blob/master/server/command/command.go#L87